cid_test.go

```diff
@@ -383,3 +373,39 @@ func TestFromJson(t *testing.T) {
 		t.Fatal("json parsing failed")
 	}
 }
+
+func BenchmarkStringV1(b *testing.B) {
+	data := []byte("this is some test content")
+	hash, _ := mh.Sum(data, mh.SHA2_256, -1)
+	cid := NewCidV1(Raw, hash)
+
+	b.ReportAllocs()
+	b.ResetTimer()
+
+	count := 0
+	for i := 0; i < b.N; i++ {
+		count += len(cid.String())
+	}
+	if count != 49*b.N {
+		b.FailNow()
+	}
+}
+
+// making sure we don't allocate when returning bytes
```

Comment:
But we do now and can't avoid it.  I take it `BenchmarkBytesV1` is leftover from the first attempt with used []byte as the representation.